live/live.default.xml

```diff
@@ -285,7 +285,22 @@
 
             </extractor>
 
-            <extractor name="org.dbpedia.extraction.mappings.AbstractExtractor" status="ACTIVE">
+            <extractor name="org.dbpedia.extraction.mappings.AbstractExtractor" status="PURGE">
```

Comment:
do you currently use the abstract extractor in live and there are `dbo:abstract` triples in the DB?
if yes, the trick would be to either implement the functionality or temporarily change the original abstract extractor to return `empty` until all pages are processed (1-2 months)
if not, you can just enable the new extractor and everything will be fine